corehq/pillows/migrations/0001_elastic_v5_mapping_compatibility.py

```diff
@@ -0,0 +1,34 @@
+from django.db import migrations
```

Comment:
As you're aware, I just added a very similar migration, though I did it in `corehq/ex-submodules/pillowtop` rather than `corehq/pillows`.  Do you have a good mental model over what belongs in the one vs the other?  Guessing pillowtop is for general, abstract stuff, and the latter is for specific individual pillows.  I suppose with that in mind, `pillows` is probably the right place for this.

Reply:
Honestly I didn't think about `pillowtop` when I did this yesterday (and I didn't realize there were existing migrations in that module).  The the mappings actually _live_ in `corehq/pillows/mappings`, so it seemed like the "right" place to put this at the time.  I completely missed the fact that your PR had it in a different location.  I modeled this (as you did, I think) after the `corehq/apps/app_manager/migrations/0007_add_linked_app_fields_to_es.py` migration, but that location doesn't seem like the correct place.